server/routes/ContactsRoutes.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+const router = express.Router();
+import {list,show,create,update,remove} from "../controllers/ContactsController";
+
+router.get("/contacts", list);
+router.get("contacts:id", show);
```

Comment:
Review the checklist for making routes. the path needs to be a very specific way
https://docs.google.com/presentation/d/1pNOnAEtE9X7vfoC5i-bINiao6j80i4x6bfocVZ3DtNw/edit#slide=id.g245a7b0889_0_92